docs/tools/flow-cli/flix.md

```diff
@@ -1,35 +1,74 @@
 sidebar_position: 12
 ---
 
-The Flow CLI provides a command to execute Flow Interaction Templates (FLIX). FLIX are an easy way to interact with Flow. They are a standard for distributing Cadence scripts and transactions, and metadata about them, in a way that is consumable by tooling and wallets. FLIX are audited for correctness and safety by an ecosystem of auditors.
+### Execute
+
+The Flow CLI provides a `flix` command to `execute` Flow Interaction Templates (FLIX). FLIX are a standard for distributing Cadence scripts and transactions, and metadata in a way that is consumable by tooling and wallets. FLIX can be audited for correctness and safety by auditors in the ecosystem.
 
 ```shell
-flow flix <query> [<argument> <argument>...] [flags]
+flow flix execute <query> [<argument> <argument>...] [flags]
 ```
 
 Queries can be a FLIX id, name, or path to a local FLIX file.
 
-## Example Usage
+### Execute Usage
 
 ```shell
 # Execute a FLIX transaction by name on Testnet
-flow flix transfer-flow 5.0 "0x123" --network testnet --signer "testnet-account"
+flow flix execute transfer-flow 5.0 "0x123" --network testnet --signer "testnet-account"
 ```
 
 ```shell
 # Execute a FLIX script by id on Testnet
-flow flix bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
+flow flix execute bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f --network testnet
 ```
 
 ```shell
 # Execute a local FLIX script by path on Testnet
-flow flix ./multiply.template.json 2 3 --network testnet
+flow flix execute ./multiply.template.json 2 3 --network testnet
+```
+
+The Flow CLI provides a `flix` command to `package` up generated plain and simple javascript. This javascript uses fcl-js to call the cadence the Flow Interaction Templates (FLIX) is based on. 
+
+<Callout type="info">
+Currently flix package command only supports generating fcl-js specific javascript, there are plans to support other languages like golang.
+</Callout>
+
+
+```shell
+flow flix package <query> [flags]
+```
+
+## Package
+
+Queries can be a FLIX id, name, or path to a local FLIX file.
+
+### Package Usage
+
+```shell
+# Generate fcl-js package code that will call the FLIX transaction cadence, save the output to a specific file
+flow flix package transfer-flow --save ./package/transfer-flow.js
+```
+
+```shell
+# Geneate package code for a FLIX script using id, since there is no saving file, the result will display in terminal
+flow flix package bd10ab0bf472e6b58ecc0398e9b3d1bd58a4205f14a7099c52c0640d9589295f 
+```
+
+```shell
+# Generate package code using local template file to save in a local file 
+flow flix package ./multiply.template.json --save ./multiply.js
 ```
 
+<Callout type="info">
+SOON: flix `generate` will generate FLIX json files using flix properties in comment blocks directly in Cadence code, [FLIP - Interaction Template Cadence Doc](https://github.com/onflow/flips/pull/80)
```

Comment:
- "flix generate" should all be in backticks
- uppercase "JSON"
- uppercase "FLIX" before properties